spec/acceptance/nodesets/el9.yml

```
@@ -0,0 +1,37 @@
+<%
+  if ENV['BEAKER_HYPERVISOR']
+    hypervisor = ENV['BEAKER_HYPERVISOR']
+  else
+    hypervisor = 'vagrant'
+  end
+-%>
+HOSTS:
+  el9:
+    roles:
+      - el9
+      - master
+      - default
+    platform: el-9-x86_64
+    box: centos/9
+    box_url: https://kojihub.stream.centos.org/kojifiles/packages/CentOS-Stream-Vagrant/9/20220417.d.0/images/CentOS-Stream-Vagrant-9-20220417.d.0.x86_64.vagrant-virtualbox.box
```

Wouldn't cloud.centos.org be a better mirror?
```suggestion
    box_url: https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-Vagrant-9-20220509.0.x86_64.vagrant-virtualbox.box
```